big_tests/tests/metrics_helper.erl

```diff
@@ -7,34 +7,9 @@
 
 %% We introduce a convention, where metrics-related suites use only 2 accounts
 %% but it depends on `all_metrics_are_global` flag, which duet it will be.
--define(WAIT_TIME, 500).
 -define(METRICS_GROUP_USERS, [alice, bob]).
 -define(ONLY_GLOBAL_METRICS_GROUP_USERS, [clusterguy, clusterbuddy]).
 
-get_counter_value(CounterName) ->
-    get_counter_value(domain_helper:host_type(mim), CounterName).
-
-get_counter_value(HostType, Metric) ->
-    case rpc(mim(), mongoose_metrics, get_metric_value, [HostType, Metric]) of
-        {ok, [{count, Total}, {one, _}]} ->
-            {value, Total};
-        {ok, [{value, Value} | _]} when is_integer(Value) ->
-            {value, Value};
-        {ok, Value} ->
-            {value, Value};
-        _ ->
-            {error, unknown_counter}
-    end.
-
-sample(Name) ->
-    rpc(mim(), mongoose_metrics, sample_metric, [[global, Name]]).
-
-assert_counter(Value, CounterName) ->
-    assert_counter(domain_helper:host_type(mim), Value, CounterName).
-
-assert_counter(HostType, Value, CounterName) ->
-    {value, Value} = get_counter_value(HostType, CounterName).
-
 -spec prepare_by_all_metrics_are_global(Config :: list(), UseAllMetricsAreGlobal :: boolean()) ->
```

Comment:
This function is still used to test the new `all_metrics_all_global` option. It doesn't need to change the config because the `all_metrics_are_global` group tests the mim2 node, which already has the `instrumentation.exometer.all_metrics_are_global` option set to true.

Reply:
Thanks for the explanation 👍